central/reports/scheduler/schedule.go

```diff
@@ -310,25 +322,28 @@ func (s *scheduler) sendReportResults(req *ReportRequest) error {
 	return nil
 }
 
-func formatMessage(rc *storage.ReportConfiguration) (string, error) {
+func getBrandedProductName() string {
```

Comment:
I think this function would be more appropriate in `pkg/branding/env-or-something.go`. The thinking is that `"RHACS_BRANDING"` definitely belongs there, and branded product name can be used somewhere else.
Plus you'd be able to test `getBrandedProductName` behavior w.r.t. environment variable as a unit isolated from the caller of `getBrandedProductName`.

Reply:
Makes sense, I'll do that.